src/Tabs.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link, useMatch } from 'react-router-dom';
+
+const tabsFromServer = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
+
+export const Tabs = () => {
+  const match = useMatch('/tabs/:tabId');
+  const selectedTabId = match?.params.tabId;
+
+  const selectedTab = tabsFromServer.find(tab => tab.id === selectedTabId);
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabsFromServer.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={tab.id === selectedTabId ? 'is-active' : ''}
+            >
+              <Link to={`/tabs/${tab.id}`}>{tab.title}</Link>
```

Comment:
you should think of readability. Multiple jsx expressions don't seem too good together
```suggestion
              <Link to={`/tabs/${tab.id}`}>
                {tab.title}
              </Link>
```